workflow/rules/common.smk

```diff
@@ -74,9 +78,15 @@ def collect_all_runs():
         run_config["model_type"] = model_type
         run_id = run_config.pop("run_id")
         runs[run_id] = run_config
+        runs[run_id] = run_config
         if model_type == "interpolator":
-            run_id = run_config["forecaster"]["run_id"]
-            runs[run_id] = run_config["forecaster"]
+            if "forecaster" not in run_config or run_config["forecaster"] is None:
+                logging.warning(
+                    f"Interpolator '{run_id}' has not been set a 'forecaster' config. Please use the 'interpolator_from_files.yaml' config to run an interpolator without a forecaster."
+                )
```

Comment:
I think this warning is superfluous. We should not warn if the interpolator is properly specified, but raise an error if not.

If the interpolator_from_files.yaml is not specified for forecasting from analysis, this will trigger an error at inference time. If we want to provide a more helpful log entry, fine, but I think otherwise we could also completely ditch this.